go/obscuronode/enclave/enclaverunner/cli.go

```diff
@@ -32,7 +34,12 @@ type EnclaveConfigToml struct {
 // ParseConfig returns a config.EnclaveConfig based on either the file identified by the `config` flag, or the flags
 // with specific defaults (if the `config` flag isn't specified).
 func ParseConfig() config.EnclaveConfig {
-	defaultConfig := config.DefaultEnclaveConfig()
+	defaultConfig := defaultconfig.DefaultEnclaveConfig()
```

Comment:
I find this name confusing. 
I would call this variable "config", and the one below :"eRC20ContractAddrs"

You start from the default as a baseline, basically make a copy of it, and then start populating the fields with what was passed in the config file.

Reply:
Sure, fixed.